apps/gro/fields/how.js

```diff
@@ -13,17 +13,20 @@ module.exports = {
       value: 'telephone',
       label: 'fields.how-radio.options.telephone.label',
       toggle: 'telephone-toggle-text',
-      child: 'input-text'
+      child: `<div id='telephone-toggle-text-panel'>
+                <div class='panel-indent'>
+                  {{#input-text}}telephone-toggle-text{{/input-text}}
+                  {{#input-text}}telephone-toggle-text-2{{/input-text}}
+                </div>
+              </div>`
     }, {
       value: 'post',
-      label: 'fields.how-radio.options.post.label',
-      toggle: 'post-toggle-text',
-      child: 'input-text'
+      label: 'fields.how-radio.options.post.label'
```

Comment:
yeah, I didn't change this line though - would be better as a separate PR to sweep and update all of them IMO